### PR TITLE
Fixed a few spelling/grammar/link issues

### PR DESCRIPTION
**Pull request by Alonski**
This could probably be improved a bit more. Not sure exactly what this does.

### CODE REVIEW

**Comment by kellyselden** [comment]
You have a legitimate failure in the tests based on the console logging.

**Comment by Alonski** [comment]
Doh... It seems that the tests check the content. The tests also need to be fixed. I will try to find time to get this done in the near future

**Comment by Alonski** [comment]
BTW this makes it harder to contribute docs changes. Not sure what to do about it. I wouldn't have expected the change I made to be tested :)

**Comment by rwjblue** [comment]
> I wouldn't have expected the change I made to be tested :)

If it were just updating the comment, I'd agree but this also updates console output which we think is fairly important to have tested...

**Comment by Alonski** [comment]
Oh double doh... Makes sense :)

File: lib/utilities/windows-admin.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ const chalk = require('chalk');
...
+   * On Windows, users will have a much better experience if symlinks are enabled
+   * and usable. When queried, this object informs Windows users regarding
+   * improving their build performance, and how.
    *
-   *  > Windows vista: nothing we can really do, so we fall back to junctions for folders + copying of files
-   *  <= Windows vista: symlinks are available but using them is somewhat tricky
-   *    * if the users is an admin, the process needed to have been started with elevated privs
+   *  > Windows Vista: nothing we can really do, so we fall back to junctions for folders + copying of files
+   *  <= Windows Vista: symlinks are available but using them is somewhat tricky
+   *    * if the user is an admin, the process needs to have been started with elevated priveleges
```
Comment (locks) [inline lib/utilities/windows-admin.js]:
```suggestion
   *    * if the user is an admin, the process needs to have been started with elevated privileges
```

**Comment by Alonski** [comment]
Awesome work @locks! Thanks

**Pull request merged**